modules/analysis-common/src/main/java/org/opensearch/analysis/common/SerbianAnalyzerProvider.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+/*
+ * Licensed to Elasticsearch under one or more contributor
```

Comment:
This is a new file so it doesn't need an Elasticsearch license. The license header above this one is sufficient.

Can you also please confirm that this is not a copy of any non-open-source-APLv2-elasticsearch code?

Reply:
I agree with your assessment and will update the file accordingly.
I am not entirely sure how to verify non-open-source-APLv2-elasticsearch code on my own and would appreciate your confirmation to ensure everything is compliant.